RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: minor
+
+Added subcategories to categories to be able to cycle through which subcategories are present for a category.
+Updated plugin to better work with modern pelican standards (by updating tasks.py and ensuring proper code formatting)
```

Comment:
I should probably add a changelog, but I don't understand this addition. Is it it meant to be converted to a changelog by some automation tool?

Reply:
I followed the guide on Pelican's website. See: https://docs.getpelican.com/en/latest/contribute.html#submitting-your-changes

Reply:
That's my fault, Oliver. This bit is related to [AutoPub](http://github.com/autopub/autopub), a tool that I built to automate GitHub + PyPI releases upon pull request merge. I had meant to add AutoPub integration to More Categories as well, but that to-do item seems to have slipped between the cracks. I will submit a separate pull request for that for you to review, which will include continuous integration (CI) for More Categories, so its tests will automatically be run on pull requests such as this one. 😄

Reply:
So should I leave this one as is for now?